contrib/julia-config.jl

```diff
@@ -11,8 +11,6 @@ const options = [
     "--framework"
 ];
 
-threadingOn() = ccall(:jl_threading_enabled, Cint, ()) != 0
```

Comment:
The function `threadingOn()` in `julia-config.jl` is the only use of `jl_threading_enabled` other than the tests just removed. It looks like `threadingOn` itself is not used anywhere in the script.